src/sentry/models/deleteduser.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import absolute_import
```

Comment:
Let's leave this entire file out for now until we figure out exactly what our plans are for this, since I'm not sure the fields we have here are what we want to collect. We can keep this to org/team/project for this PR.